src/utils/config.py

```diff
@@ -81,10 +81,12 @@ def reload(self, force_reload=False):
         workspace_url = os.environ.get('WS_URL', kbase_endpoint + '/ws')
         catalog_url = os.environ.get('CATALOG_URL', kbase_endpoint + '/catalog')
         re_api_url = os.environ.get('RE_URL', kbase_endpoint + '/relation_engine_api').strip('/')
-        service_wizard_url = os.environ.get('SW_URL', kbase_endpoint + '/service_wizard').strip('/')
-        sample_service_release = os.environ.get('SAMPLE_SERVICE_RELEASE', 'dev')
-        sample_service_url = get_sample_service_url(service_wizard_url, sample_service_release)
-        config_url = os.environ['GLOBAL_CONFIG_URL']
+        sample_service_url = os.environ.get("SAMPLE_SERVICE_URL")
```

Comment:
Mostly for testing and dev, allow setting the sampleservice url directly and bypass the extra call to the service wizard